lib/client-assets.php

```diff
@@ -1068,12 +1068,22 @@ function gutenberg_editor_scripts_and_styles( $hook ) {
 		sprintf( '/wp/v2/users/me?post_type=%s&context=edit', $post_type ),
 	);
 
+	// Ensure the global $post remains the same after
+	// API data is preloaded. Because API preloading
+	// can call the_content and other filters, callbacks
+	// can unexpectedly modify $post and cause the sky
```

Comment:
"The sky to fall on Gutenberg's head" is definitely a cute turn-of-phrase, but it might confuse ESL folks or someone not familiar with what that means–and itself doesn't really add any info. I will push a comment tweak, but if you can add any additional context feel free 😄

Reply:
Works for me!